### PR TITLE
Doc: update user guide for Pre-Launched RT mode

### PR DESCRIPTION
**Pull request by taoyuhong**
The Release_2.1 use multiboot2, so update the example

Signed-off-by: Tao Yuhong <yuhong.tao@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/pre-launched-rt.rst [hunk 1]
```diff
@@ -106,3 +105,14 @@ Reboot the system, and it will boot into Pre-Launched RT Mode
    26c5e0d88f8a47d88109f201ebd61a5e   0   ACRN PRE-LAUNCHED VM0            Running
    dbbbd4347a574216a12c2201f1ab0240   1   ACRN SOS VM                      Running
    ACRN:\>
+
+Connect console of VM0, via 'vm_console' ACRN shell command (Use 'Ctrl + Alt + Space' key
```
Comment (dbkinder) [inline doc/tutorials/pre-launched-rt.rst:109]: can you verify if this should be CTRL + SPACE  (no ALT needed)?

File: doc/tutorials/pre-launched-rt.rst [hunk 2]
```diff
@@ -76,7 +76,7 @@ ethernet 03:00.0 devices to the Pre-Launched RT VM, build ACRN with:
    make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/whl-ipc-i5.xml SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml
```
Comment (zhanqigithub) [inline doc/tutorials/pre-launched-rt.rst:76]: add RELEASE=0 build parameter

File: doc/tutorials/pre-launched-rt.rst [hunk 3]
```diff
@@ -88,12 +88,11 @@ like this:
...
 
-   menuentry 'ACRN Hybrid' {
-       set root=hd1,gpt1
-       echo 'loading hybrid...'
-       multiboot --quirk-modules-after-kernel /EFI/BOOT/acrn.32.out root=/dev/nvme0n1p3
-       module /EFI/BOOT/bzImage_RT RT_bzImage
-       module /EFI/BOOT/bzImage Linux_bzImage
+   menuentry 'ACRN multiboot2 hybrid'{
+       echo 'loading multiboot2 hybrid...'
+       multiboot2 /EFI/BOOT/acrn.bin
```
[the remark this answers is not in the document]
Comment (taoyuhong) [reply doc/tutorials/pre-launched-rt.rst:93]: Have tested, do not need that. All VM kernel cmdlines are in hybrid_rt.xml. And that is confusing, that is multiboot kernel param, why that is only work in SOS, how to explain that?

File: doc/tutorials/pre-launched-rt.rst (hunk 2 above)
Comment (taoyuhong) [reply doc/tutorials/pre-launched-rt.rst:76]: OK

File: doc/tutorials/pre-launched-rt.rst (hunk 1 above)
Comment (taoyuhong) [reply doc/tutorials/pre-launched-rt.rst:109]: (CTRL + SPACE) is right

Comment (taoyuhong) [reply doc/tutorials/pre-launched-rt.rst]: Fixed

File: doc/tutorials/pre-launched-rt.rst (hunk 2 above)
Comment (taoyuhong) [reply doc/tutorials/pre-launched-rt.rst]: added

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**